src/App.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import store from './redux/store';
```

Comment:
## [OPTIONAL]

- [ ] Kindly make sure the your app is using the API correctly, you should not create an new API every time the app is refreshed, kindly it need to be created once 

```jsx
// the below code should only have bein only used once, the data in the API should not not desapear when the page is refreshed,
import axios from 'axios';

const id = 'TUBThKZKOHdsLlkMqwTD';

const api = axios.create({
  baseURL: `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${id}`,
});

export default api;
```

- [ ] KIndly make sure also the 3 books that was use to intiate the state are removed, they are not neeeded as we have already an API to store our books.

- [ ] As the API is having a field to hold the auther of the book, I would sugget adding it also, as is importent for the Library recored